pkg/assembler/backends/ent/backend/builders_test.go

```diff
@@ -251,3 +261,31 @@ func (s *Suite) TestExactBuilder() {
 		})
 	}
 }
+
+func (s *Suite) TestMixedInputsIngestionBuilders() {
```

Comment:
This test fully makes sense :+1: 
I think for consistency with other tests, this should be added to `TestIngestBuilder` as `Ingest same twice` for ingesting the same twice and then check the query just returns one.